Hm, tests are failing with `test.zig:41:65: error: expected type 'manifest.Manifest', found 'test.man__struct_19051'`, but it looks like this is happening on `main` too - any idea what that might be?

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        zig_version: ["master"] # eventually use multiple versions once stable
+        zig_version: ["0.13.0"] # eventually use multiple versions once stable
         rust:
           - stable
     steps:

--- a/src/compiled_plugin.zig
+++ b/src/compiled_plugin.zig
@@ -30,7 +30,7 @@ pub fn init(allocator: std.mem.Allocator, data: []const u8, functions: []const F
 
     if (plugin == null) {
         // TODO: figure out what to do with this error
-        std.debug.print("extism_compiled_plugin_new: {s}", .{
+        std.debug.print("extism_compiled_plugin_new: {s}\n", .{
             errmsg,
         });
         c.extism_plugin_new_error_free(errmsg);

--- a/src/plugin.zig
+++ b/src/plugin.zig
@@ -56,7 +56,7 @@ pub fn initFromCompiled(compiled: *CompiledPlugin) !Self {
     const plugin = c.extism_plugin_new_from_compiled(compiled.ptr, &errmsg);
     if (plugin == null) {
         // TODO: figure out what to do with this error
-        std.debug.print("extism_plugin_new: {s}", .{
+        std.debug.print("extism_plugin_new: {s}\n", .{
             errmsg,
         });
         c.extism_plugin_new_error_free(errmsg);

